docker.go

```diff
@@ -35,11 +35,19 @@ func WriteDockerfile(builder *strings.Builder, node *parser.Node, useOriginal bo
 	if useOriginal {
 		builder.WriteString(node.Original)
 	} else {
-		builder.WriteString(node.Value)
+		splits := strings.Split(node.Value, "  ")
+		splitsTrimmed := []string{}
+		for _, split := range splits {
+			if split != "" {
+				splitsTrimmed = append(splitsTrimmed, strings.TrimSpace(split))
+			}
+		}
+		s := strings.Join(splitsTrimmed, " \\ \n\t")
```

Comment:
This will cause a tab character instead of whitespace. Typically in `Dockerfiles`, whitespaces are used.

Though I'm not sure if we should use two or four spaces. I think four is typically used (going by the original `Dockerfile`)

Reply:
The yellow is `\t` and the red is a trailing whitespace
<img width="847" alt="image" src="https://github.com/SongStitch/anchor/assets/856001/f1866bd8-ef9f-4679-99d9-571f6ce08eb3">

Reply:
Yeah I was thinking about that. I'll switch to 4 spaces and remove this extra space here.